src/ucp/rma/rma_sw.c

```diff
@@ -338,14 +338,11 @@ static void ucp_rma_sw_dump_packet(ucp_worker_h worker, uct_am_trace_type_t type
                      length - header_len);
 }
 
-UCP_DEFINE_AM(UCP_FEATURE_RMA, UCP_AM_ID_PUT, ucp_put_handler,
-              ucp_rma_sw_dump_packet, 0);
-UCP_DEFINE_AM(UCP_FEATURE_RMA, UCP_AM_ID_GET_REQ, ucp_get_req_handler,
-              ucp_rma_sw_dump_packet, 0);
+UCP_DEFINE_AM_PROXY(UCP_FEATURE_RMA, UCP_AM_ID_PUT, ucp_put_handler,
```

Comment:
can we fully replace UCP_DEFINE_AM and not define another macro UCP_DEFINE_AM_PROXY?

Reply:
UCP_DEFINE_AM_PROXY adds proxy handler function. do you mean add such proxy always?

Reply:
I mean UCP_DEFINE_AM_PROXY defines proxy function, UCP_DEFINE_AM - not. do you mean add proxy into every AM handler?

Reply:
before this PR, we used UCP_DEFINE_AM macro for all AM handlers. Why not keep doing so (even if we had proxy for every AM)?